ac1/_sources/regression_and_line_of_best_fit/introduction.rst

```diff
@@ -0,0 +1,40 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+.. _introduction:
+
+Introduction
+============
+
+In the last section, you learned that correlation in scatter plots measures the
+linear relationship between two quantitative variables. The closer the
+coefficient of determination (or *r* value) is to 1, and the closer the points
```

Comment:
In the file above, you say R^2 is the coefficient of determination, but here you say it is r.